go.mod

```diff
@@ -3,6 +3,8 @@ module knative.dev/kn-plugin-event
 go 1.16
 
 require (
+	github.com/Azure/go-autorest/autorest v0.11.17 // indirect
+	github.com/Azure/go-autorest/autorest/adal v0.9.10 // indirect
```

Comment:
I don't know why this is poping here, but I think we need to accept this change.